pkg/tsdb/cloudwatch/request_parser.go

```diff
@@ -166,6 +166,11 @@ func parseRequestQuery(model *simplejson.Json, refId string, startTime time.Time
 		metricEditorModeValue = metricEditorMode(memv)
 	}
 
+	fillMissingMode := fillMissing{
```

Comment:
@andresmgot We talked earlier about checking the query type before trying to get this information here.
However, sometimes it seems that query type is undefined and still executed as a time series query. Like [here](https://github.com/grafana/grafana/blob/main/pkg/tsdb/cloudwatch/cloudwatch.go#L294). 
[Here](https://github.com/grafana/grafana/blob/main/pkg/tsdb/cloudwatch/request_parser.go#L150) we also don't worry too much if the query type isn't defined (although maybe that contradicts its caller [here](https://github.com/grafana/grafana/blob/main/pkg/tsdb/cloudwatch/request_parser.go#L32)?)

I think it would be safest not to restrict checking the json keys by query type and instead rely on the zero value (which is now the default `null` fill mode since I am not using the plugin-sdk-go definition anymore).

Let me know what you think.